VideoLocker/build.gradle

```diff
@@ -180,6 +182,7 @@ android {
         // test configuration
         testApplicationId 'org.edx.mobile.test'
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner "com.facebook.testing.screenshot.ScreenshotTestRunner"
```

Comment:
So oddly enough, the screenshot tests are unable to use the @Test tag which I'm assuming it means its using Junit3 test runner. The E2E tests are using Junit4 which allows the @Test tag so we ended up getting two separate test suites on accident.